views/game.html.erb

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" class="mdl-js">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="assets/favicon.ico" rel="icon">
+    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
+    <link href="assets/css/styles.css" rel="stylesheet">
+    <script src="assets/js/jquery.min.js"></script>
+    <script src="assets/js/bootstrap.min.js"></script>
+    <title>Codebreaker Web</title>
+  <body>
+    <section id="introduction">
+      <div class="container">
+        <div class="row fill-viewport align-items-center justify-content-center">
+          <div class="col-12 col-md-6">
+            <div class="row">
+              <div class="col-md-auto">
+                <h1 class="text-white title">Codebreaker 2018</h1>
+              </div>
+            </div>
+            <p class="lead text-white-70">
+              Hello, test!
+              Try to guess 4-digit number, that consists of numbers in a range from 1 to 6.
+            </p>
+            <hr class="mb-4">
+            <%= render_partial('./views/shared/_info-panel.html.erb') %>
+            <hr class="mb-4">
+            <%= render_partial('./views/shared/_errors.html.erb') %>
+            <form method="post" id=guess action="/game">
+              <button type="button" disabled class="btn btn-primary float-right">
+                Secret code:
+                <span class="badge badge-light"><%= secret_code %></span>
+              </button>
+              <div class="form-group">
+                <input class="form-control guess_input" name="guess" pattern="^[1-6]{4}$" placeholder="1234" required="" autocomplete="off">
+              </div>
+              <div class="form-group guess_markers_list">
+                <div id="guess_marker">
+                   <% Array.new(4 - last_comparison.length) {'x'}.prepend(*last_comparison.split('')).each do |sign| %>
+                      <%=
+                        case sign
+                        when '+' then '<button type="button" class="btn btn-success marks" disabled>+</button>'
+                        when '-' then '<button type="button" class="btn btn-primary marks" disabled>-</button>'
+                        else '<button type="button" class="btn btn-danger marks" disabled>x</button>'
+                        end
+                      %>
+                    <% end %>
```

Comment:
It would be better to move it to helper method or create a decorator.

Reply:
yes, helper is good idea for this code